### PR TITLE
Merged master into loginPage

### PR DESCRIPTION
**Pull request by JinHash5**

### CODE REVIEW

**Comment by krishamehta** [comment]
@ikbalsingh please review.

File: src/Components/Forms.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    render(){
+        const isRegister = this.state.isRegister;
+        let button = null;
+        if(this.state.isRegister){
+            button = <div id="reg">
+                <input type="text" placeholder="First Name" value={this.state.firstName} onChange={this.handlefirstName} required/>
+                <input type="text" placeholder="Last Name" value={this.state.lastName} onChange={this.handlelastName} required/>
+                <input type="text" placeholder="SAP ID" value={this.state.sapID} onChange={this.handleSAP} required/>
+                <input type="text" placeholder="Mobile Number" value={this.state.mobileNumber} onChange={this.handlemobileNumber} required/>
+
```
Comment (ikbalsingh) [inline src/Components/Forms.js:123]: Shouldn't the Mobile Number and SAP ID fields be type = "number"?

File: src/Components/Forms.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+                            Login
+                        </button>
+                        </center>
+                </form>
+                <center>
+                    <button id="register" className="btn btn-info btn-block login" style={{width:'100%', backgroundColor: '#e7998f'}} onClick={this.addRegister} disabled={this.state.isRegister}>
+                        Register
+                        </button>
+                </center>
+            </div>
```
Comment (ikbalsingh) [inline src/Components/Forms.js:150]: Is there some reason of putting the Register button outside the form element?

**Comment by ikbalsingh** [comment]
Also, the placeholder text is not visible due to the background. Please change the color so as to make it more visible.

**Comment by AvaisP** [comment]
Add .idea to gitignore

**Review by ikbalsingh — APPROVED** [review]

**Pull request merged**